addons/website_event_booth/static/src/scss/website_event_booth.scss

```diff
@@ -71,3 +71,54 @@ label.o_wbooth_category_unavailable {
 .o_wevent_booth_category {
     background-color: $o-wevent-bg-color-light;
 }
+
+.o_wevent_booth_progress_bar {
+    $wevent-booth-progress-bar-color: mix(map-get($theme-colors, 'primary'), $body-bg, 40%);
+    $wevent-booth-progress-bar-color-2: mix(map-get($theme-colors, 'primary'), $body-bg, 33%);
+    height: 48px;
+    li {
+        border: 1px solid $wevent-booth-progress-bar-color;
+        border-right-width: 0;
+        padding-left: 24px;
+        height: 48px;
+        line-height: 48px;
+        width: 33%;
```

Comment:
If we want to avoid any rounding issue for high res screens and have an unique size, we could go with a simpler calc(100%/3) , as it seems it is used in other places, actually.